maintain_pim/common_issues/index.rst

```diff
@@ -104,39 +104,13 @@ Elasticsearch defines a ``index.mapping.total_fields.limit`` `parameter <https:/
        "status":400
     }
 
-To do so, create a ``my_index_configuration.yml`` file with the following content:
 
-.. code-block:: yaml
+To do so, you must update the ``APP_ELASTICSEARCH_TOTAL_FIELDS_LIMIT`` environment variable. (see https://docs.akeneo.com/4.0/migrate_pim/upgrade/upgrade_from_32_to_40.html#local-or-on-premise-environment)
 
-    settings:
-        mapping:
-            total_fields:
-                limit: 12000 # fix your own limit here
-
-Then, load the ``my_index_configuration.yml`` by adding it to the Symfony ``elasticsearch_index_configuration_files`` parameter that is present in the file ``parameters.yml`` or ``pim_parameters.yml``.
-
-For instance, if you have a *Community* edition:
-
-.. code-block:: yaml
-
-    # parameters.yml
-    parameters:
-        # ...
-        elasticsearch_index_configuration_files:
-            - '%kernel.root_dir%/../vendor/akeneo/pim-community-dev/src/Akeneo/Pim/Enrichment/Bundle/Resources/elasticsearch/index_configuration.yml'
-            - '/path/to/my_index_configuration.yml'
-
-If you have the *Enterprise* edition:
+.. code-block:: bash
 
-.. code-block:: yaml
+    APP_ELASTICSEARCH_TOTAL_FIELDS_LIMIT=12000
```

Comment:
could you remove the documentation about `akeneo_pim_product` curl request and `akeneo_pim_product_model` curl request please also? It's just below, in the next paragraph.

These indexes don't exist anymore. (just keep `curl -XPUT 'localhost:9200/akeneo_pim_product_and_product_model/_settings`)